packages/grafana-runtime/src/utils/userStorage.tsx

```diff
@@ -156,3 +156,26 @@ export function usePluginUserStorage(): PluginUserStorage {
   }
   return new UserStorage(context?.meta.id);
 }
+
+/**
+ * A function for interacting with the backend user storage (or local storage if not enabled).
+ * @returns An scoped object for a plugin and a user with getItem and setItem functions.
+ * @alpha Experimental
+ */
+export function getPluginUserStorage(): UserStorage {
```

Comment:
Since this only works for apps, I'd call this `getAppPluginUserStorage`

Also I am not sure if this should throw an error or return `null` instead if the user storage cannot be initiated. I believe that'd be better and it would be up to the caller if it wants to throw an error (and crash the app) or have some fallback mechanism.

Reply:
The difference is that in the function above, the `context` should always be available (so not having it is a real error). In this case, the plugin ID _may_ not be present in the URL, and that's a valid situation so throwing an error in that case may not be the best option.